app/controllers/ops_controller.rb

```diff
@@ -741,16 +741,14 @@ def rebuild_toolbars(presenter)
     presenter[:record_id] = determine_record_id_for_presenter
   end
 
-  def handle_bottom_cell(nodetype, presenter, r, locals)
+  def handle_bottom_cell(presenter, r, locals)
     # Handle bottom cell
-    if nodetype == "log_depot_edit"
-      presenter.hide(:form_buttons_div, :pc_div_1)
-    elsif @pages || @in_a_form
+    if @pages || @in_a_form
       if @pages
         presenter.hide(:form_buttons_div).show(:pc_div_1)
         presenter.update(:paging_div, r[:partial => "layouts/x_pagingcontrols"])
       elsif @in_a_form
-        presenter.update(:form_buttons_div, r[:partial => "layouts/x_edit_buttons", :locals => locals])
+        presenter.update(:form_buttons_div, r[:partial => "layouts/angular/paging_div_buttons"])
```

Comment:
A question.. are all the forms in ops angular already? Because if not, they will have broken buttons, right?